### DISCUSSION TITLE
How to reduce size of marker-end on Edge.

### DISCUSSION THREAD

**Discussion by mueen-memon** [xyflow/xyflow#2842] [Q&A]
Hi, I am trying to reduce the size of arrow. It is increasing and decreasing based on the size of edge stroke.  
<img width="534" alt="image" src="https://user-images.githubusercontent.com/89274259/218682200-92b5b6d7-0eb3-4b0f-9a49-129e86e9bf04.png">

I have tried setting markerEnd{width and height} but its offset is not right.
<img width="546" alt="image" src="https://user-images.githubusercontent.com/89274259/218683126-82235a42-ac6a-4795-80e0-19c36c5a3cbf.png">

**Comment by CamboimGabriel** [comment]
I think is better you create a custom marker and then call it inside reactflow! that way you can adjust the offset you want or even the shape!
